File: python/Sequencial/EX10_B.py
#10.faca um programa que calcule e mostre a area de um circulo.
# Sabe-se que: Area = pi=Rpontencia de 2
# 49085.938
import math

print("***********************SEQUÊNCIAL*****************************")
print("**********************Programa*10B****************************")
print("***************Calculo da area do Circulo*********************")

raio =  float(input("Qual a media do raio: "))

area = math.pi * (raio * raio)
print('O tamanho do circulo é: %5.3f' % area)

File: JS/Sequencial/EX10_B.js
/*
10.faca um programa que calcule e mostre a area de um circulo.
Sabe-se que: Area = pi=Rpontencia de 2
*/

confirm("Calcular a área do circulo")

var raio = prompt("Digite a medida do raio")
raio = parseFloat(raio)

area = Math.PI * (raio * raio)

console.log("A area do circulo é igual a: "+ area.toFixed(3))